### ISSUE TITLE
The program '[17020] Phoesion.Glow.Reactor.Debugger.exe' has exited with code 4294967295 (0xffffffff).

### ISSUE THREAD

**Issue by adaskalopoulos-yodiwo** [Phoesion/Glow#3]
I tried the "Hello World" service - https://glow-docs.phoesion.com/tutorials/Create_Hello_World.html. When pressing F5 at Visual Studio I get:

The program '[17020] Phoesion.Glow.Reactor.Debugger.exe' has exited with code 4294967295 (0xffffffff).

I am expecting to see a window of the service running.

I am using latest release - 0.0.299.

**Comment by gepa21** [comment]
Hello,

I was able to reproduce and identity the issue in a problematic escaping of path between Visual Studio and Windows.

the command line that is trying to launch the application is something like : 
` Phoesion.Glow.Reactor.Debugger.exe -QuantumSpace:Default "bin\Debug\net6.0\win-x64\Foompany.Services.HelloWorld.dll"`

that _(for some reason)_ the `\n` string in the path for `Debug*\n*et6.0` gets escaped _(it shouldn't)_ and ends up as a new line in the arguments! _(The debugger then fails to find the file and cannot load it)_

I will release a new version in couple of days that will take this into account and de-escape new lines into the "\n" string, which should fix the problem.

In the meantime, a workaround for your project is to add the 
`<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>`
property in your .csproj file, so the output does not contain a directory start with the letter n _(in this case net6.0)_

so your project file should be something like : 
```
<PropertyGroup>
   <TargetFramework>net6.0</TargetFramework>
   <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
</PropertyGroup>
```

Thank you for reporting this issue!

**Comment by gepa21** [comment]
fixed in version [v0.0.300](https://glow-docs.phoesion.com/release_notes/index.html#release-v00300)
